Formula/ghi.rb

```diff
@@ -1,10 +1,9 @@
 class Ghi < Formula
   desc "Work on GitHub issues on the command-line"
   homepage "https://github.com/stephencelis/ghi"
```

Comment:
```suggestion
  homepage "https://github.com/drazisil/ghi"
```

Reply:
Did this fork get endorsed?

Reply:
It's listed on the previous homepage https://github.com/stephencelis/ghi
> `ghi` was originally created by [Stephen Celis](https://github.com/stephencelis), and is now maintained by [drazisil](https://github.com/drazisil)'s fork [here](https://github.com/drazisil/ghi).